tools/vuln_valid/index.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+const Joi = require('joi');
```

Comment:
Can you remove the hapi'isms from the code style (capitalized imports, four space indentation, empty line at function start, etc.).